company-keywords.el

```diff
@@ -317,3 +337,4 @@
 
 (provide 'company-keywords)
 ;;; company-keywords.el ends here
+
```

Comment:
Please drop the addition of the last empty line.

Reply:
Sorry for this.